docs/animations/Light.md

```diff
@@ -98,7 +98,7 @@ Await MyUIElement.Light(distance:=5, duration:=2500, delay:=250, color:=Colors.R
 
```

Comment:
We actually removed this functionality. @Sergio0694 I don't think we have an equivalent yet, eh? We should just put a note at the top or something maybe for now? (I'm hoping we can do an experiment in our new Labs space to replace this in the future, but it's not high on the priority list at the moment.)

Reply:
Yeah we don't have light helpers currently. Not entirely sure why we removed them 🤔

Reply:
I believe it was a catch-22 with how we wanted to re-architect the behaviors/media packages or something, and they were the only things remaining that was hard to migrate or something (iirc). The underlying Windows API I believe was deprecated and replaced with something new (but harder to use). So we've had a similar proposal like what we did with Shadows open to resolve it. Something to do in Labs that's been on my list for a while, but just hasn't met the bar yet.